### ISSUE TITLE
Specifying SPACK_MANAGER in environment

### ISSUE THREAD

**Issue by tasmith4** [sandialabs/spack-manager#163]
I was trying to use the new reduced workflow commands as follows, starting from empty directory `spack-manager-test`, in a fresh shell:
```
spack-manager-test$ git clone --recursive git@github.com:psakievich/spack-manager
# a bunch of git output, spack-manager is checked out at 9c2fd5e
spack-manager-test$ source spack-manager/scripts/useful_bash_functions.sh
spack-manager-test$ quick-develop
+ spack-start
-bash: /start.sh: No such file or directory

ERROR: Exiting quick-develop prematurely
```

This error goes away if I run `export SPACK_MANAGER=$(pwd)/spack-manager` -- is the fact that the useful bash functions fail without that environment variable a workflow feature or bug?

If it's considered a feature, I'd like to request all these functions fail immediately if SPACK_MANAGER is unset with a clear message asking the user to set it.

**Comment by psakievich** [comment]
good catch.  `spack-start` has a check for this but it is in `start.sh` 
🤦‍♂️

**Comment by tasmith4** [comment]
I think the auto-close didn't work, but I've tested on latest main and this looks to be resolved now -- thanks!

**Comment by tasmith4** [comment]
@psakievich I spoke too soon.  Originally, I accidentally tested in a shell where I had already set SPACK_MANAGER.  In a fresh shell, it does give a friendly error message, but also drops me off the remote machine:
```
$ source spack-manager/start.sh
Env variable SPACK_MANAGER not set. You must set this variable.
Connection to ascicgpu057 closed.
```
I'd prefer if it didn't exit my remote shell. :)  Perhaps there is a simple bash tweak to fix this?

**Comment by psakievich** [comment]
Yeah that fix was in the PR we closed. I'll add a fix in #184 . @tasmith4

**Comment by psakievich** [comment]
@tasmith4 this should have been addressed in #184. Closing for now, but please reopen if you run into any more issues
